src/responseType.ts

```diff
@@ -3,9 +3,17 @@ type BaseError = {
   message: string;
 };
 
-type FieldError<RequestData> = BaseError & { property: keyof RequestData };
+type Path<ObjectType extends object> = {
```

Comment:
```suggestion
type Path<ObjectType extends Record<string, unknown>> = {
```
I think it's discouraged to use the `object` type

Reply:
I need to check if it still works with Record. Somehow I am not getting any linter/type-checker warnings about this anymore. I'll check if we disabled the rule or if things have changed.

Reply:
It turns out that `object` is correct here and `Record<string, unknown>` is causing some issues (too deep type declaration).